src/main/java/reactor/core/publisher/WorkQueueProcessor.java

```diff
@@ -65,16 +65,151 @@
 public final class WorkQueueProcessor<E> extends EventLoopProcessor<E> {
 
 	/**
+	 * {@link WorkQueueProcessor} builder that can be used to create new
+	 * processors.
+	 *
+	 * @param <T> Type of dispatched signal
+	 */
+	public final static class Builder<T> {
+
+		String name;
+		ExecutorService executor;
+		ExecutorService requestTaskExecutor;
+		int bufferSize;
+		WaitStrategy waitStrategy;
+		boolean share;
+		boolean autoCancel;
+
+		/**
+		 * Creates a new {@link WorkQueueProcessor} builder with default properties.
+		 * @return new WorkQueueProcessor builder
+		 */
+		public static <T> Builder<T> create()  {
+			return new Builder<T>();
+		}
+
+		Builder() {
+			this.name = WorkQueueProcessor.class.getSimpleName();
```

Comment:
as in the other processor, name() and waitStrategy() accept nulls but I don't think the ctor does, so the default should maybe be applied at `build()` time.